### ISSUE TITLE
Maps of variables

### ISSUE THREAD

**Issue by MichaelSchulzMETNO** [metno/pyaerocom#522]
The maps of non-observable maps, such aerosol speciation of AOD, emissions, loads and deposition can be very useful for characterising the model version. Good to have standard ranges for the scales as in idl.

**Comment by MichaelSchulzMETNO** [comment]
The map function could be to the advantage also enhanced by:
- be by default available for CMAS84 experiments (missing now it seems @jgriesfeller ?)
- be also available per year, season averages

**Comment by avaldebe** [comment]
GeoJSON files would be too big for piloting on the (spatial) original resolution.
We would need an API that servers the data, or the plots directly.

**Comment by jgriesfeller** [comment]
In my opinion we need to think about the maps again. For my work (C3S2), maps that are not in the original model resolution are useless. I don't need daily maps or an animation. Monthly, seasonal, and yearly averages are needed though.

Technically, I don't think that GeoJSON (basically a text file) is the right format to store that data. In addition, I don't want a contour plot of the data. I want to see the individual pixels instead. Without investigation, I think this is a job for [WMS](https://en.wikipedia.org/wiki/Web_Map_Service) (This is a transfer protocol e.g. used for uEMEP visualisation) or [GeoTiff](https://en.wikipedia.org/wiki/GeoTIFF)

**Comment by AugustinMortier** [comment]
I would be convinced that the current implementation is useless (for your work) when you show me the you don't seen any differences with the current method :)

.. considering that the polygons are computed from the original resolution, so it should reflect original patterns considering the number of layers used to compute the polygons. The 5x5 grid is mostly used for computing the time series dynamically (even though we make it possible to visualize it), which make possible to compute time series on the flow for custom defined regions. So if we want to keep this feature, we would still need to compute this 5x5 file, as long as we do not have a rest API that does the job.

![image](https://user-images.githubusercontent.com/19610522/142878806-21f96227-f7b0-40f8-be12-5adfd64daf49.png)

**Comment by jgriesfeller** [comment]
The main point for the original resolution is that these plots have always been in the original resolution. I need the map plots for reports and my "customers" just want to see them in original resolution.
In addition, satellite data has small spacial data gaps, small spacial extremes etc all of which are not visible anymore when decreasing resolution to 5 degrees. And don't forget that these maps are supposed to be used for regional model evaluation. Do you really want to show a map of 7km model resolution degraded to 5 degrees?

So in short: I need a map plot that I can put directly into a project report (Word file). The current one does not deliver that. The new features are nice, but since they can't be put in a pdf, not needed by me. 
Sorry, but you have sacrificed usefulness for look.

**Comment by MichaelSchulzMETNO** [comment]
Usefull is a relative term! But let's discuss this in person...

**Comment by AugustinMortier** [comment]
I don't know if you read my comment (which would facilitate the discussion) but the polygons (default view) are produced with the original model resolution. So all the 5 degrees comments are not relevant here. Only the time resolution is computed from the 5x5 regridded models.

In the case of the aerocom experiment run by Jonas, the polygons make things actually easier to interpret as it highlights areas with different behaviors. Just open two models simultaneously and see how different things are. 
And if the details would not be high enough in other cases, we could always increase the number of layers to see more patterns.. that's why I want you to try out this feature to see if this is really useless for you.. considering that the feature is already implemented!

Now if you have the need for a single grid plot to be inserted directly in a report, yes we probably would have to use WMS. But this should be decided together considering all the projects and the needs of everyone.

**Comment by jgriesfeller** [comment]
Here's an example from DOMOS that I need a little help with. I used the standard parameters for the maps and I did find the grid view now.
Polygon view:
![polygon view](https://user-images.githubusercontent.com/10772065/148908744-ffc2257a-c2e3-4e58-bf39-49311edbc765.png)

Grid view:
![grid view](https://user-images.githubusercontent.com/10772065/148908791-b11eaf4f-3fa4-4e7f-a797-82d7a2535e89.png)

The Polygon view obviously did not use the data from the grid view. Any suggestions about how to improve?

The data is [here:](https://aeroval.met.no/maps.php?project=domos&exp_name=SLSTR+dust+retrievals)
